zaas-service/src/main/resources/zaas-log-messages.yml

```diff
@@ -23,27 +23,6 @@ messages:
     # Service specific messages
     # 700-999
 
```

Comment:
Cleanup of this file (messages not used in ZAAS) will be part of another task I suppose

Reply:
a future PR